_test/test_tobyfit/test_refine_crystal/test_tobyfit_refine_crystal_1.m

```diff
@@ -68,6 +68,15 @@ function test_tobyfit_refine_crystal_1 (option)
 % File to which to save results of refinement
 savefile='test_tobyfit_refine_crystal_1_out.mat';   % filename where saved results are written
 
+test_tobyfit_dir = fullfile(horace_root(), '_test', 'test_tobyfit');
```

Comment:
this identical block code is in four files...

Reply:
The `onCleanup()` statement can't be refactored into another function (as far as I know). I'll move the other two lines into a function